src/Compiler/Scheme/Racket.idr

```diff
@@ -330,42 +335,48 @@ getFgnCall : {auto f : Ref Done (List String) } ->
 getFgnCall appdir (n, fc, d) = schFgnDef appdir fc n d
 
 startRacket : String -> String -> String -> String
-startRacket racket appdir target = unlines
-    [ "#!/bin/sh"
-    , "# " ++ (generatedString "Racket")
-    , ""
-    , "set -e # exit on any error"
-    , ""
-    , "if [ \"$(uname)\" = Darwin ]; then"
-    , "  DIR=$(zsh -c 'printf %s \"$0:A:h\"' \"$0\")"
-    , "else"
-    , "  DIR=$(dirname \"$(readlink -f -- \"$0\")\")"
-    , "fi"
-    , ""
-    , "export LD_LIBRARY_PATH=\"$DIR/" ++ appdir ++ ":$LD_LIBRARY_PATH\""
-    , "export DYLD_LIBRARY_PATH=\"$DIR/" ++ appdir ++ ":$DYLD_LIBRARY_PATH\""
-    , racket ++ "\"$DIR/" ++ target ++ "\" \"$@\""
-    ]
+startRacket racket appdir target = """
+  #!/bin/sh
+  # \{ generatedString "Racket" }
+
+  set -e # exit on any error
+
+  if [ "$(uname)" = Darwin ]; then
+    DIR=$(zsh -c 'printf %s "$0:A:h"' "$0")
+  else
+    DIR=$(dirname "$(readlink -f -- "$0")")
+  fi
+
+  export LD_LIBRARY_PATH="$DIR/\{ appdir }:$LD_LIBRARY_PATH"
+  export DYLD_LIBRARY_PATH="$DIR/\{ appdir }:$DYLD_LIBRARY_PATH"
+
+  \{ racket } "$DIR/\{ target }" "$@"
+  """
 
 startRacketCmd : String -> String -> String -> String
-startRacketCmd racket appdir target = unlines
-    [ "@echo off"
-    , "set APPDIR=%~dp0"
-    , "set PATH=%APPDIR%" ++ appdir ++ ";%PATH%"
-    , racket ++ "\"%APPDIR%" ++ target ++ "\" %*"
-    ]
+startRacketCmd racket appdir target = """
+  @echo off
+
+  rem \{ generatedString "Racket" }
```

Comment:
Is this line extra?

Reply:
Yup

Reply:
should it be removed? I'm not clear why it's been added

Reply:
Are we talking about the vertical whitespace or the comment with a function that inserts CG's name?

If it is the latter, all shell startup wrapper scripts have this function that inserts CG's name, so I added it here for consistency.

Reply:
It was the latter, seems good to me!

Reply:
@andrevidela Thanks for the review!